sklearn/decomposition/_nmf.py

```diff
@@ -900,10 +1014,27 @@ def non_negative_factorization(X, W=None, H=None, n_components=None, *,
         If init='custom', it is used as initial guess for the solution.
         If update_H=False, it is used as a constant, to solve for W only.
 
+    A : array-like of shape (n_components, n_features), default=None
+        Initial guess for the numerator auxiliary function, only used in
+        :class:`sklearn.decomposition.MiniBatchNMF`.
+
+        .. versionadded:: 0.XX
+
+    B : array-like of shape (n_components, n_features), default=None
+        Initial guess for the denominator auxiliary function, only used in
+        :class:`sklearn.decomposition.MiniBatchNMF`.
+
+        .. versionadded:: 0.XX
+
     n_components : int, default=None
         Number of components, if n_components is not set all features
         are kept.
 
+    batch_size : int, default=None
+        Number of samples per batch: only for MiniBatch implementation.
```

Comment:
Instead of saying it is only *for* the minibatch implementation, we should say that using `batch_size != None` is the way to *select* the minibatch implementation.